ServiceWebsite/ServiceWebsite/ClientApp/src/translate/messages.xlf

```diff
@@ -997,25 +1000,45 @@
           <context context-type="linenumber">15</context>
         </context-group>
       </trans-unit>
+      <trans-unit id="differentHearingType_h1_1" datatype="html">
+        <source>Which type of court hearing?</source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">app/modules/individual-journey/pages/different-hearing-types/different-hearing-types.component.html</context>
+          <context context-type="linenumber">2</context>
+        </context-group>
+      </trans-unit>
       <trans-unit id="differentHearingType_p_1" datatype="html">
         <source>
-    The court will decide which method is suitable for your hearing. You can help the court decide by
-    answering some questions.
+      The court will decide if it&apos;s more suitable for the hearing to take place in a court building or by video.
   </source>
         <context-group purpose="location">
           <context context-type="sourcefile">app/modules/individual-journey/pages/different-hearing-types/different-hearing-types.component.html</context>
-          <context context-type="linenumber">2</context>
+          <context context-type="linenumber">3</context>
         </context-group>
+      </trans-unit>
+      <trans-unit id="differentHearingType_p_2" datatype="html">
+        <source>
+      We need to ask you some questions to help the court decide.
+    </source>
         <context-group purpose="location">
           <context context-type="sourcefile">app/modules/individual-journey/pages/different-hearing-types/different-hearing-types.component.html</context>
-          <context context-type="linenumber">8</context>
+          <context context-type="linenumber">7</context>
+        </context-group>
+      </trans-unit>
+      <trans-unit id="differentHearingType_p_3" datatype="html">
+        <source>
+      Before you answer the questions, we&apos;ll show you what the different types of hearing are like.
+  </source>
+        <context-group purpose="location">
+          <context context-type="sourcefile">app/modules/individual-journey/pages/different-hearing-types/different-hearing-types.component.html</context>
+          <context context-type="linenumber">11</context>
         </context-group>
       </trans-unit>
       <trans-unit id="journey_btn_continue" datatype="html">
         <source>Continue</source>
         <context-group purpose="location">
           <context context-type="sourcefile">app/modules/individual-journey/pages/different-hearing-types/different-hearing-types.component.html</context>
-          <context context-type="linenumber">12</context>
+          <context context-type="linenumber">15</context>
```

Comment:
We were agreed to remove this file? and don't update it

Reply:
As discussed we will are not going to commit this file